test/cpp/end2end/async_end2end_test.cc

```diff
@@ -263,8 +263,9 @@ class AsyncEnd2endTest : public ::testing::TestWithParam<TestScenario> {
     void* ignored_tag;
     bool ignored_ok;
     cq_->Shutdown();
-    while (cq_->Next(&ignored_tag, &ignored_ok))
+    while (cq_->Next(&ignored_tag, &ignored_ok)) {
       ;
```

Comment:
No need for this anymore, now that there are braces around it.

Reply:
Done